pygenn/genn_model.py

```diff
@@ -69,15 +69,16 @@ class GeNNModel(object):
     """
 
     def __init__(self, precision=None, model_name="GeNNModel",
-                 enable_debug=False, cpu_only=False):
+                 enable_debug=False, cpu=None):
```

Comment:
Is it not "cpu_only= None"?

Reply:
Oh - I see ... just also renamed it everywhere. I don't mind. We called it "cpu_only" in the past because one can of course also use the cpu in the GPU/CPU version.

Reply:
This was intentional - ``CPU_ONLY`` is returned by the wrapper to signify that there's no GPU support, here you're saying whether you want to use the CPU or not (if that makes sense)

Reply:
I see now - very sensible ... you could of course choose to use the CPU even if GPU support is available.